tests/bsim/bluetooth/audio/src/mcc_test.c

```diff
@@ -2462,7 +2463,10 @@ void test_main(void)
 		bt_addr_le_to_str(bt_conn_get_dst(default_conn), addr, sizeof(addr));
 		printk("Connected: %s\n", addr);
 
-		bt_conn_le_param_update(default_conn, BT_LE_CONN_PARAM(0x06U, 0x10U, 0U, 400U));
+		bt_conn_le_param_update(default_conn,
+					BT_LE_CONN_PARAM(BT_GAP_MS_TO_CONN_INT(7.5),
+							 BT_GAP_MS_TO_CONN_INT(10), 0U,
```

Comment:
I'm pretty sure that this line is correct, but do note that `BT_GAP_MS_TO_CONN_INT(10)` evaluates to 0x08 and not 0x10 as in the original code.